packages/terra-functional-testing/tests/wdio/run-axe-spec.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * This spec is used for testing the global axe command.
+ */
+
+describe('Axe', () => {
+  it('should successfully run axe on the page', () => {
+    browser.url('https://engineering.cerner.com/terra-ui/');
+
+    const { result } = browser.axe();
```

Comment:
Can we return result object directly instead of returning an object with the result object?

Reply:
I think we'll want to keep this as is unless we shift error handling into the the `run-axe` command. I was thinking about adding the error handling into the calling commands (the chai assertions/validates) and keeping the axe run command as a pass through:

https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#context-parameter-examples

I'm not really sure under what circumstances this will throw an error, but we'll want to catch it and log it/fail the assertion somewhere in the stack. I don't think consumers should be running this command directly, I think they should be running the assertions/validation commands

Reply:
hmm yeah It'd be some things to think about. I've been thinking about the current axe setup and it would be nice if we could wrap these in try-catch to try to prevent the ugly axe errors we sometimes see. I'm not sure if we could catch those or not depending on where those are being throw.